RELEASES.md

```diff
@@ -1,6 +1,7 @@
 Version 0.9.2 (2023-03-XX)
 ========================
 * Draw MPC path instead of model predicted path, this is a more accurate representation of what the car will do.
+* Honda HR-V Vezel 2023 non US support thanks to Alex Takeda!
```

Comment:
I would imagine the US version will be similar, so let's remove the non-US for now unless we know that they differ. You can treat this car as if location doesn't matter (rename in docs, platform, etc)